### PR TITLE
feature/#4770/Implement notification in user profile cabinet 

### PR DESCRIPTION
**Pull request by SokhaTaras**

### CODE REVIEW

**Review by AngelinaRudnitska — APPROVED** [review]

**Review by vplvua — APPROVED** [review]

File: src/app/ubs/ubs-user/ubs-user-profile-page/ubs-user-profile-page.component.ts [hunk 1]
```diff
@@ -153,6 +159,16 @@ export class UbsUserProfilePageComponent implements OnInit {
     this.isEditing = false;
   }
 
+  checkedCheckbox(event) {
+    if (event.source.id === 'telegramNotification') {
+      this.telegramNotification = event.checked ? true : false;
```
Comment (hnativlyubomyr) [inline src/app/ubs/ubs-user/ubs-user-profile-page/ubs-user-profile-page.component.ts]: this.telegramNotification = event.checked

File: src/app/ubs/ubs-user/ubs-user-profile-page/ubs-user-profile-page.component.ts [hunk 2]
```diff
@@ -153,6 +159,16 @@ export class UbsUserProfilePageComponent implements OnInit {
...
+    }
+
+    if (event.source.id === 'viberNotification') {
+      this.viberNotification = event.checked ? true : false;
```
Comment (hnativlyubomyr) [inline src/app/ubs/ubs-user/ubs-user-profile-page/ubs-user-profile-page.component.ts]: this.viberNotification = event.checked

File: src/app/ubs/ubs-user/ubs-user-profile-page/ubs-user-profile-page.component.ts [hunk 3]
```diff
@@ -193,18 +211,43 @@ export class UbsUserProfilePageComponent implements OnInit {
...
         }
       );
       this.alternativeEmailDisplay = false;
+      this.redirectToMessengers();
     } else {
       this.isEditing = true;
     }
   }
 
+  redirectToMessengers() {
```
Comment (hnativlyubomyr) [inline src/app/ubs/ubs-user/ubs-user-profile-page/ubs-user-profile-page.component.ts:229]: update this method
redirectToMessengers() {
if (this.telegramNotification) {
        this.goToTelegramUrl();
      }
      if (this.viberNotification) {
        this.goToViberUrl();
      }
}

Comment (SokhaTaras) [reply src/app/ubs/ubs-user/ubs-user-profile-page/ubs-user-profile-page.component.ts:229]: Yeah,thanks

**Review by atsa21 — APPROVED** [review]

**Review by zaitsev-oleksii — APPROVED** [review]

**Review by marianaharnyk — APPROVED** [review]

**Review by hnativlyubomyr — APPROVED** [review]

**Pull request merged**